projects/canopy/src/lib/button/button.notes.ts

```diff
@@ -56,8 +56,10 @@ The \`iconButton\` property is used to visually hide that text.
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
-| \`\`variant\`\` | The variant of button: \`\`solid-primary\`\`, \`\`solid-secondary\`\`, \`\`outline-primary\`\`, \`\`outline-secondary\`\`, \`\`reverse-primary\`\`, \`\`reverse-secondary\`\`; | string | primary | No |
+| \`\`variant\`\` | The variant of button: \`\`solid-primary\`\`, \`\`solid-secondary\`\`, \`\`outline-primary\`\`, \`\`outline-secondary\`\`, \`\`reverse-primary\`\`, \`\`reverse-secondary\`\`, \`\`add-on\`\`; | string | solid-primary | No |
+| \`\`size\`\` | The size of the button | ButtonSize [\`\`sm\`\`, \`\`md\`\`] | \`\`md\`\` | No |
 | \`\`fullWidth\`\` | If the button has to span full width or not. For 'sm' and 'md' sized screens, the button will always be full width and this input has no affect | boolean | false | No |
+| \`\`disabled\`\` | Programmatically disable the button via this property | boolean | false | No |
```

Comment:
I probably missed something but why do we need a `disabled` input when a user can simply add the attribute to the html button element itself?

Reply:
The reason I added it on this is that if you disable the form control and there is a button suffix it will automatically disable the button. It's much easier to do this programmatically via the component instance as opposed to going into the markup to do it. I can see that it does provide two options to do the same thing though, not sure if there is a cleaner alternative.